Assets/Scripts/AndroidPermissionsManager.cs

```diff
@@ -1,19 +1,42 @@
+using System;
 using UnityEngine;
 
 public class AndroidPermissionCallback : AndroidJavaProxy
 {
-    public AndroidPermissionCallback() : base("com.unity3d.player.UnityAndroidPermissions$IPermissionRequestResult") { }
+	private event Action<string> OnPermissionGrantedAction;
```

Comment:
Just that you know: https://stackoverflow.blog/2017/06/15/developers-use-spaces-make-money-use-tabs/ :)